package.json

```diff
@@ -28,9 +28,8 @@
     "lint:css": "stylelint './src/**/*.ts'"
   },
   "dependencies": {
-    "@tippyjs/react": "^4.0.0-alpha.4",
-    "downshift": "^3.4.8",
-    "tippy.js": "^6.1.1"
```

Comment:
Do we not need this? We are still importing the css from it in Tooltip.jsx

Reply:
When I play around with it in storybook and in playground, it doesn't have any issues

Reply:
the original tippy is a dependency of tippyjs react anyway